docs/index.rst

```diff
@@ -46,5 +72,42 @@ reuse already existing properties file since no changes were introduced there)
 
     pytest tests/foreman/ui_airgun/test_architecture.py
 
+
+Contributing
+------------
+
+Good practice when working with Python project as AirGun is to work in
+Python's virtual environments. To initiate and activate them before running
+`pip ...` commands run:
+
+.. code-block:: bash
+
+    virtualenv venv   # on Fedora you might need to use `virtualenv-3` if you have Python 2 as a default on your system
+    source venv/bin/activate
+
+If you are about to start contributing, please follow `GitHub flow`_.
+
+For a new-comer, we have issues tagged with "`good first issue`_" so that
+might be place where you can start?
+
+When testing your AirGun code using Robottelo tests, I have used following
+ugly trick to have mine current development version of AirGun available in
+Robottelo virtual environment:
+
+.. code-block:: bash
+
+    mv venv/lib/python3.6/site-packages/airgun{,ORIG}
+    ln -s ../airgun/airgun/ venv/lib/python3.6/site-packages/airgun   # depends where you have airgun checked out
+
+When you are running your tests locally, you (currently) need Chrome browser
+installed and `chromedriver`_ binary available somewhere in your PATH (in
+your robottelo checkout): download it, unzip it and put e.g. into (if you
+are using virtual environments) `venv/bin/`.
```

Comment:
Not necessary, there's `webdriver_binary=` option in robottelo.properties where you can specify path to chromedriver. Only if it's not specified then robottelo will look for it in PATH.

Reply:
Oh, and maybe it's worth mentioning explicitly, that the only supported webdriver for now is chromedriver.